src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+}
+
+.container_item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  flex-grow: 1;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+
+}
+
+.item_1 {
+  background-color: hsl(0, 100%, 50%);
+}
+
+.item_2 {
+  background-color: hsl(0, 100%, 40%);
+}
```

```suggestion
.container_item: nth-child(2) {
  background-color: hsl(0, 100%, 40%);
}
```
Давай використаємо nth-child замість купи класів